server/db/models/Review.js

```diff
@@ -1,12 +1,24 @@
-const db = require('../db');
-const Sequelize = require('sequelize');
-//MODEL
-const Review = db.define('review', {
+const db = require("../db");
+const Sequelize = require("sequelize");
+
+const Review = db.define("review", {
+  id: {
+    primaryKey: true,
+    type: Sequelize.UUID,
+    defaultValue: Sequelize.UUIDV4
+  },
+  title: {
+    type: Sequelize.STRING(200),
+    allowNull: false
+  },
+  author: {
+    type: Sequelize.STRING
+  },
   body: {
     type: Sequelize.TEXT,
     allowNull: false,
     validate: {
-      len: [10,250]
+      len: [10, 500]
```

Comment:
How about we don't impose a limit on this? Reviews should be allows to be expressive I think. This isn't Twitter...

Reply:
You want them longer? Thanks cool - I'll take the limit off and have it set to type.text on the next merge